src/pages/invest.tsx

```diff
@@ -0,0 +1,16 @@
+import Layout from "@components/Layout";
+import React, { ReactElement } from "react";
+
+function Invest() {
+	return <div className="px-28 py-11">Invest page</div>;
```

Comment:
this can become 
```
<Layout.WithMainMenu>
<Layout.WithSideMenu>
<div className="px-28 py-11">Invest page</div>;
</Layout.WithSideMenu>
</Layout.WithMainMenu>
```